File: src/ApproximatorTester/ApproximatorTester.Tests/test/data/AllTests/FourVars.cs
{caret}namespace N
{
    class Program
    {
        static void Main(string[] args)
        {
            string query1 = "Select";
            const string field = " fieldName ";
            query1 += "from";
            var tableName = "Table1";
            if (args.Length != 0)
            {
                query1 += tableName;
            }
            else
            {
                tableName = "Table2";
                query1 += tableName;
            }
            string filter = "where field is numeric";
            query1 += filter;
            Program.ExecuteImmediate(query1);
        }

        static void ExecuteImmediate(string query)
        {
            
        }
    }
}

File: src/SELApproximator/SELApproximator.Tests/test/data/JsFsa/FourVars.js
{caret}function main(cond)
{
    var query1 = "a";
    var field = " not used ";
    query1 += "b";
    var tableName = "c";
    if (cond)
    {
        query1 += tableName;
    }
    else
    {
        tableName = "d";
        query1 += tableName;
    }
    var filter = "e";
    query1 += filter;
    execScript(query1);
}